CHANGELOG.md

```diff
@@ -1,13 +1,20 @@
-## 0.9.26 (TBD, 2020)
+## 1.0.0-rc1 (TBD, 2020)
```

Comment:
Should we maybe go with something like `0.99.0` instead to technically stick within SemVer format?

Reply:
By my interpretation of https://semver.org/#spec-item-9 it is a valid SemVer format.

Reply:
Ok cool, I wasn't aware of that.  Sounds good.